### ISSUE TITLE
Wired Antenna

### ISSUE THREAD

**Issue by henriquefchaves** [mv4wd/RCHourglass#5]
Are you still working and available for this project? I have few questions about the spiral wired antenna in the transponder.
I only get wire with different diameters, has like an equation to adapt the antenna?

Thanks in advance

**Comment by henriquefchaves** [comment]
One more thing, components at loop amplifier circuit from J5 until C5 are not used in the project? Because the circuit is open
![Screenshot 2021-09-07 171017](https://user-images.githubusercontent.com/25842660/132377281-7f5f4971-2504-491a-baba-70a621828827.png)

**Comment by mv4wd** [comment]
The are some  online calculators of coil but the result are always aproximations, so a small change in wire diameter should not be a big issue. For the jumpers, the need to be shorted when smaller timing loop is used: 9'x1' to 12'x1' : No jumpers on loop amplifier.

6'x1' to 8'x1' : Use jumper at J3 on loop amplifier.

3'x1' to 4'x1' : Use jumper at J5 on loop amplifier.

**Comment by henriquefchaves** [comment]
Thanks for the response!
Soo, can you tell me what is the frequency for that square wire antenna? ( just for curiosity and to undertsand more of the project)
About the cables used for the detecting loop, they need to have some specific features or it can be any cable like single-wire cable 0.4mm for connections?
![2021-09-14_14-15](https://user-images.githubusercontent.com/25842660/133264374-a46c79d5-69d1-45a7-b87f-fb56c52a47db.png)



sorry, one more question.
The dimensions that you mentioned above "9'x1' to 12'x1'" correspond to what in the project?? Again the detecting loop area?

**Comment by mv4wd** [comment]
The resonance of the circuit should be tuned around 5MHz. For the loop wire I've always used 18 awg wire. The dimension are of the loop on the ground. Usually I don't have any jumper

**Comment by henriquefchaves** [comment]
I upload the firmware version01.hex file and my blue led is always off since then, also i can't connect to the psoc UART, no device is recognized in my PC.
Do you have any ideia of waht can happen?
I bought 2 controller's already.

**Comment by henriquefchaves** [comment]
@mv4wd  When in the wiki you say to connect the mini usb cable to the PSOC, at the same time the other side (normal USB to program firmware) has to be connected to? or only the mini usb?

**Comment by henriquefchaves** [comment]
@mv4wd  i am trying to update the code **version01_beta.hex**, but when i do it, i cant connect the uart from the micro-usb to the PC. I tried other codes from mysefl and the uart works with them. Can you help me?

**Comment by mv4wd** [comment]
Have you connected all the components? without the crystal on the board the PSOC will not turn on

**Comment by henriquefchaves** [comment]
You are saying that we need to connect the psoc with the circuit daughter board? Is not optional?
I connected to decoder and the 2 capacitors and Cristal

**Comment by mv4wd** [comment]
Yes, crystal and 2 capacitors need to be there. Also check that you have installed Cypress serial drivers (does at least the PSOC enumerate as unknown device?). Some people had issue with some USB cables (but you say other project works)

**Comment by henriquefchaves** [comment]
Thankss verry muchh! I think it worked now.

**Comment by henriquefchaves** [comment]
@mv4wd we are with some problems finishing the project.. Is there way to test the transponder? More specifically the antenna?
In the transponder, it is suppost to have electrical contact between the terminal's of the wired antenna (antenna loop)?
Another thing, on the loop amplifier we have 1.8V in the base of Q6 (suppost to be 2.2) and 2.4V on Q3 collector, is that supose to be equal? Or do you think this is a big problem?
We used bc547 and 557 on loop amplifier too. Can be a problem?

**Comment by mv4wd** [comment]
Something wrong with those reading. Are you powering it trough 75 ohm resistor with a 5 volt power supply as written in the wiki? Is input power around 4.18 V at the input (so it means after 75ohm resistor running to +5) Beware for BJT pinout, double check the datasheets. For the transponder antenna I don't understand the question. The loop is a plain wire so il will read as short circuit when connected.

**Comment by WaldoBorek** [comment]
In case of loop amplifier there is ready to buy box which is working very good with decoder from mv4wd.
It is called MLA30+. Price on aliexpress ~40 euro.
Much better signal/reception. 

![image](https://user-images.githubusercontent.com/48533358/134351643-9adc6e50-2176-45b0-83a6-4420447e5c65.png)

**Comment by henriquefchaves** [comment]
Okk, thanks a lot.
We already check pinout BJT and is correct. But do you think using BC547 and BC557 on loop amplifier can be the problem?

**Comment by mv4wd** [comment]
No, I don't think. It's the correct complementary pair. I've used BC547 and BC307 because that's what I've found at my local shop. But if you want to be 100% sure, stick with the original Cano design, ie 2N3904 & 2N3906. Start to check from higher voltages 5v, then 4.18 after 75 ohm resistance etc. Too low voltages mean too much current draw, and that might upset the whole design

**Comment by malcx95** [comment]
> In case of loop amplifier there is ready to buy box which is working very good with decoder from mv4wd. It is called MLA30+. Price on aliexpress ~40 euro. Much better signal/reception.
> 
> ![image](https://user-images.githubusercontent.com/48533358/134351643-9adc6e50-2176-45b0-83a6-4420447e5c65.png)

Hi, did you modify anything in the circuit in order to make it work with this antenna? Did you use it to completely replace the loop amplifier? Did you just use the antenna part or did you also use the amplifier part of the MLA30+?

**Comment by mv4wd** [comment]
Never used the MLA-30, sorry. I have friends who did and found it helpful to improve detection. I didn't check out the exact connection, but I guess you need to bypass the psoc power to the antenna and just use its own power supply and a cap decoupler. A diode protection at PSOC input could be a good idea since antenna power is higher than PSOC input max voltage allowed
